packages/backend/src/api.ts

```diff
@@ -23,6 +24,11 @@ const traceAPICall: express.RequestHandler = (request, _, next) => {
   next();
 };
 
+const routeValidator = new OrbitAPIValidator({
+  allowUnsupportedRoute: true,
+  mutateWithDefaultValues: true,
```

Comment:
allows request queries and bodies to be _augmented_ with the default values defined in the JSON schema